e2e/logInSignUp.spec.js

```diff
@@ -134,8 +135,22 @@ describe('Log in flow', () => {
 
 describe('testing swiping on feed page', () => {
```

Comment:
Better naming. I though this would swipe up and down posts, but it seems to swipe on the different pages